### PR TITLE
i18n support

### PR DESCRIPTION
**Pull request by RecoX**
This pull request introduces several changes to improve the localization functionality of the project, including adding support for JSON-based language files and updating the code to use these files. The most important changes include the addition of new JSON language files, modifications to handle localization settings, and updates to use JSON-based messages.

Localization improvements:

* Added new JSON language files for English and Spanish with localization messages (`Languages/1.json`, `Languages/english.json`, `Languages/spanish.json`). [[1]](diffhunk://#diff-c095816dc41d76343121e4679e7617b5f474b5b2549e82ac564ad70f21e49274R1-R3) [[2]](diffhunk://#diff-03b15982ff5fabf886db90a3511cfd94f63a32128078943cd243a73415bd60d0R1-R3)
* Updated `SetLanguageApplication` function to handle localization settings, load the appropriate JSON language file, and parse its content (`CODIGO/ModLanguage.bas`).
* Introduced a new function `DirLanguage` to return the path to the language files directory (`CODIGO/General.bas`).

Code updates to use JSON-based messages:

* Replaced hardcoded message `MENSAJE_CRIATURA_MATADO` with its JSON-based equivalent in `HandleNPCKillUser` (`CODIGO/Protocol.bas`).
* Added a new module reference for JSON handling (`Argentum20.vbp`).
* Added Microsoft ActiveX Data Objects 2.5 Library and Microsoft Scripting Runtime

Note:
Fix the Enum so we can use Spanish.json instead of 1.json

### CODE REVIEW

File: CODIGO/General.bas [hunk 1]
```diff
@@ -221,6 +221,21 @@ DirMapas_Err:
     
 End Function
 
+Public Function DirLanguage() As String
```
Comment (morgolock) [inline CODIGO/General.bas:224]: Mover esta funcion  a CODIGO/ModLanguage.bas asi tenemos todo lo de localizacion en un unico modulo

File: Argentum20.vbp [hunk 2]
```diff
@@ -3,6 +3,8 @@ Reference=*\G{00020430-0000-0000-C000-000000000046}#2.0#0#..\..\Windows\SysWOW64
 Reference=*\G{E1211242-8E94-11D1-8808-00C04FC2C603}#1.0#0#..\..\Windows\SysWow64\dx8vb.dll#DirectX 8 for Visual Basic Type Library
 Reference=*\G{56A868B0-0AD4-11CE-B03A-0020AF0BA770}#1.0#0#..\..\Windows\System32\quartz.dll#ActiveMovie control type library
 Reference=*\G{88D89F7C-BB59-4F13-9850-93CBD874039D}#1.5#0#..\Aurora.Network\build\Code\Proxy\Release\Aurora.Network.dll#Aurora.Network - Aurora Engine - Network
+Reference=*\G{420B2830-E718-11CF-893D-00A0C9054228}#1.0#0
```
Comment (morgolock) [inline Argentum20.vbp:6]: Que es esto? Una referencia a un control? Podemos eliminar los cambios en lineas 6 y 7 ?

**Review by morgolock — CHANGES_REQUESTED** [review]

File: Argentum20.vbp (hunk 2 above)
Comment (RecoX) [reply Argentum20.vbp:6]: Son estas 2 librerias

Microsoft ActiveX Data Objects 2.5 Library aca no recuerdo que utiliza

Microsoft Scripting Runtime de aca utiliza el type Dictionary

Son necesarias para poder hacer andar JSON.bas

**Review by morgolock — APPROVED** [review]

File: Languages/1.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "MENSAJE_CRIATURA_MATADO": "La criatura te ha matado!!!",
+  "LOGIN_SCREEN_WELCOME_MESSAGE": "Â¡Bienvenido de nuevo! Â¡Disfruta tu viaje por Argentum Online!"
```
Comment (RecoX) [inline Languages/1.json:3]: Hay caracteres rotos

Comment (morgolock) [reply Languages/1.json]: Arreglado en el ultimo parche

**Review by morgolock — APPROVED** [review]

**Pull request merged**